spec/controllers/items_controller_spec.rb

```diff
@@ -67,23 +67,31 @@
       end
 
       it 'redirects to root and flashes a confirmation notice when successful' do
-        post 'purge_object', params: { id: @pid }
```

Comment:
it seems like these tests should have been failing before since purge was a GET ... I don't think the verbs are being respected here by rspec

Reply:
I think this is one of the reasons that controller tests are deprecated and request specs are preferred now.